Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:edge
```

Comment:
Could you be a bit more specific? Also, should I set codeclimate as a maintainer?